prod-devel-rcar.yaml

```diff
@@ -307,6 +310,22 @@ components:
       additional_deps:
         - "../android_kernel/out/android12-5.4/common/arch/arm64/boot/Image"
 
+  zephyr:
+    build-dir: "zephyr"
+    sources:
+      - type: west
+        # `url`, `rev` and `file` are optional items.
```

Comment:
I don't think we need this. It is already covered either in Mouling or West documentation.

Reply:
Thanks, I removed the whole commented block. It looks cleaner.